ceph/rbd/workflows/rbd_python.py

```diff
@@ -0,0 +1,60 @@
+import sys
+
+import rados
+import rbd
+from docopt import docopt
+
+doc = """
+A simple test suite wrapper that executes tests based on yaml test configuration
+
+ Usage:
+  rbd_python.py --pool <name>
+        (--image <name>)
+        (--image-size <size>)
+        [--conf-file FILE]
+        [--data-pool <name>]
+
+Options:
+  -h --help                         show this screen
+  --pool <pool>                     pool name
+  --image <image>                   image name
+  --image-size <size>               image size
+  --conf-file <file>                conf file path
+  --data-pool <name>                data pool for ec pool
+"""
+
+
+def rbd_python(args):
+    """
+    Create image and write random data to it, read data written and verify
+    """
+    conf_file = args.get("--conf-file", "/etc/ceph/ceph.conf")
+    pool = args.get("--pool")
+    image_name = args.get("--image")
+    image_size = int(args.get("--image-size"))
+    data_pool = args.get("--data-pool")
+
+    with rados.Rados(conffile=conf_file) as cluster:
```

Comment:
please add comment for what we are doing here?

Reply:
Done.